src/main/java/life/qbic/model/Configuration.java

```diff
@@ -0,0 +1,13 @@
+package life.qbic.model;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Global configuration container
+ */
+public class Configuration {
+
+    public static final long MAX_DOWNLOAD_ATTEMPTS = 3;
+    public static final Path LOG_PATH = Paths.get(System.getProperty("user.dir"),"logs");
+}
```

Comment:
Does this save the logs into the applications directory or the user directory (e.g. afriedrich)?

Reply:
user.dir is the current directory